### PR TITLE
Supporting RenderItems being added Outside the RenderSpatialTree volume

### PR DESCRIPTION
**Pull request by samcake**
Fix the problemoccuring when adding a  REnderItem which bound crosses any of the limits of the Render octreee.
THe renderItem are added to the root in this case

To Test:
Go further than the 6 planes at 16384.0 meters in any direction from the origin and your avatar should still be rendered correctly.

An object Inside the octree volume (within [-16384.0, 16384.0]) but big enough to extend OUT of the octree should still be rendered correctly

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 24d712b7b4c703beabae47b901e63c249f560a3f available at: https://gist.github.com/dd70dba773990577f90b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7319.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7319.dmg

File: interface/src/ui/overlays/Base3DOverlay.cpp [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 #include "Base3DOverlay.h"
 
+#include "Application.h"
```
Comment (jherico) [inline interface/src/ui/overlays/Base3DOverlay.cpp]: why?

Comment (samcake) [inline interface/src/ui/overlays/Base3DOverlay.cpp]: To get access to qapp and the the main window

Comment (samcake) [inline interface/src/ui/overlays/Base3DOverlay.cpp]: I mean the scene

**Comment by ZappoMan** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 24d712b7b4c703beabae47b901e63c249f560a3f available at: https://gist.github.com/dd70dba773990577f90b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7319.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7319.dmg

**Comment by ZappoMan** [comment]
Build failed...

```
SpatialTree.cpp:425, GNU C Compiler 4 (gcc), Priority: Normal
variable ‘cellLocation’ set but not used [-Wunused-but-set-variable]
```

**Comment by hifi-gustavo** [comment]
Build console output for commit: 729890342e5139304d2b4ddcdf2e13542ef5f045 available at: https://gist.github.com/dd70dba773990577f90b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7319.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7319.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8281e1bc8ffabe4637494af132065d302b335220 available at: https://gist.github.com/dd70dba773990577f90b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7319.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7319.dmg

**Pull request merged**